packages/core/test/unit/utils/utils.ts

```diff
@@ -38,7 +38,25 @@ export function getNewPluginManager(plugins: string[], pluginsContext: PluginCon
   return new PluginManager(fileConfig, plugins, pluginsContext);
 }
 
-export function getNewSiteLinkManager() {
+class SiteLinkManagerMock extends SiteLinkManager {
```

Comment:
This is not really a Mock class by the definition of Mock right?

Reply:
Emm, I see; thanks for pointing this out;
I have come up with a list of potential names for it, but not sure whether "EnhancedSiteLinkManager" is the best choice. "TestableSiteLinkManager", "UtilitySiteLinkManager", "TestHelperSiteLinkManager" and  "AccessibleSiteLinkManager" are some other choices.  What is your idea?

Reply:
TestableSiteLinkManager sounds good to me!

Reply:
Done in latest commit.